invokeai/frontend/web/src/common/components/FocusRegionWrapper.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, type BoxProps } from '@invoke-ai/ui-library';
+import { useAppSelector } from 'app/store/storeHooks';
+import { type FocusRegionName, useFocusRegion, useIsRegionFocused } from 'common/hooks/focus';
+import { selectSystemShouldEnableHighlightFocusedRegions } from 'features/system/store/systemSlice';
+import { forwardRef, type RefObject, useMemo, useRef } from 'react';
+
+interface FocusRegionWrapperProps extends BoxProps {
+  region: FocusRegionName;
+  focusOnMount?: boolean;
+  highlightColor?: string;
```

Comment:
I don't see any component using this prop and would expect the highlight color to be a high-level design decision. Let's remove this prop.